rj_gameplay/rj_gameplay/tactic/goalie_tactic.py

```diff
@@ -106,9 +106,9 @@ def get_requests(self, world_state: rc.WorldState,
         # TODO: this calculation is copy-pasted from wall_tactic
         # put into common param file: https://www.geeksforgeeks.org/global-keyword-in-python/
 
-        # dist is slightly greater than penalty box bounds
-        box_w = world_state.field.penalty_long_dist_m
-        box_h = world_state.field.penalty_short_dist_m
+        # dist is slightly greater than def_area box bounds
+        box_w = world_state.field.def_area_long_dist_m
```

Comment:
This works so the problem doesn't lie in how I changed the Field class in stp/rc.py